frontends/qt/setup.nsi

```diff
@@ -103,6 +104,12 @@ Section -Main SEC0000
     #File /r /x Makefile* @abs_top_srcdir@/doc\*.*
     SetOutPath $INSTDIR
     WriteRegStr HKCU "${REGKEY}\Components" Main 1
+
+    # Explicitly copy the icon file to the installation directory
+    File "${ICONDIR}\icon.ico"
```

Comment:
Seems like this is not needed anymore, right?

Reply:
Correct, forgot to remove it in last update. Fixed now.